.github/workflows/dkg-perf.yaml

```diff
@@ -0,0 +1,31 @@
+name: DKG Run
+
+on: 
+    workflow_dispatch:
```

Comment:
what do we think about adding a repository trigger with a relay url arg? Then could potentially automate tests after a dev relay deploy or things like that

Reply:
Can do that way, i was thinking to handle the relay url though .env, as we might need to adjust other params too like the cluster size, node nums etc. wdyt?

Reply:
I guess that there could be certain things unset in the dkg env file, and then we set them in the workflow instead. Potentially starting with only `CHARON_P2P_RELAYS` could be a good tester.

Reply:
Yes, we do that way, let me send it in the next push